lib/install-panel.coffee

```diff
@@ -63,13 +63,14 @@ class InstallPanel extends View
     @loadFeaturedPackages()
 
   detached: ->
+    @packageManagerSubscription.dispose()
     @unsubscribe()
 
   focus: ->
     @searchEditorView.focus()
 
   handleSearchEvents: ->
-    @subscribe @packageManager, 'package-install-failed', (pack, error) =>
+    @packageManagerSubscription = @packageManager.on 'package-install-failed', (pack, error) =>
       @searchErrors.append(new ErrorView(@packageManager, error))
 
     @subscribe atom.commands.add @searchEditorView.element, 'core:confirm', =>
```

Comment:
So both libs are interchangeable? I thought it would be needed to first migrate the `@searchEditorView` - that is not provided by `settings-view` - to `event-kit` before replacing this.